app/components/mandatarissen/fractie-selector.js

```diff
@@ -19,27 +21,34 @@ export default class MandatenbeheerFractieSelectorComponent extends Component {
         o.get('id')
       );
     }
+    this.searchByName.perform();
   }
 
   @restartableTask
   *searchByName(searchData) {
-    yield timeout(300);
+    yield timeout(SEARCH_TIMEOUT);
     let queryParams = {
       sort: 'naam',
       include: 'fractietype',
       filter: {
-        naam: searchData,
         'bestuursorganen-in-tijd': {
           id: this.bestuursorganenId.join(','),
         },
       },
     };
+    if (searchData) {
+      queryParams.filter.naam = searchData;
+    }
     let fracties = yield this.store.query('fractie', queryParams);
     fracties = fracties.filter((f) => !f.get('fractietype.isOnafhankelijk'));
     //sets dummy
-    if ('onafhankelijk'.includes(searchData.toLowerCase())) {
```

Comment:
This whole isOnafhankelijk logic seems broken. If I understand it correctly a new fractie should be created if the person is onafhankelijk. But this is not the case at all, this only happens if you search for onafhankelijk. Which seems to give more than one fractie btw. And even then this fractie is never saved.